gen/calc.py

```diff
@@ -326,6 +326,13 @@ def calculate_config_yaml(user_arguments):
         prefix='  ' * 3)
 
 
+def calculate_mesos_isolation(enable_gpu_isolation):
+    isolators = 'cgroups/cpu,cgroups/mem,disk/du,network/cni,filesystem/linux,docker/runtime,docker/volume',
```

Comment:
I accidentally left a trailing comma here which caused this to be a tuple rather than a string, if you remove the comma the errors should go away